amgrss_schemas/amgrss_master.xml

```diff
@@ -83,7 +84,7 @@
             <amg:longDescription>Silas discovers something about Tara. The Majestic and Agrestic merger leaves some people without jobs. Shane installs a security system. Nancy discovers she's being tailed.</amg:longDescription>
             <amg:longDescription amg:lang="http://www.ebu.ch/metadata/ontologies/skos/ebu_Iso639_1LanguageCodeCS#_fr">Silas découvre quelque chose sur Tara. La fusion Majestic et Agrestic laisse certaines personnes sans emploi. Shane installe un système de sécurité. Nancy découvre qu'elle est filée.</amg:longDescription>
             <media:description>Silas discovers something about Tara. The Majestic and Agrestic merger leaves some people without jobs. Shane installs a security system. Nancy discovers she's being tailed.</media:description>
-            <media:content url="s3://ovp-destination-886239521314/5cb9f349-5375-4c91-b355-a6c06ec5535f/mp4/LG_WEEDS_311_HD_8CH_EN_Lionsgate_20210618T120642_Surround_QVBR-9_1920x1080_5495.mp4" medium="video" duration="1502" framerate="23.97"/>
+            <media:content url="s3://ovp-destination-886239521314/5cb9f349-5375-4c91-b355-a6c06ec5535f/mp4/LG_WEEDS_311_HD_8CH_EN_Lionsgate_20210618T120642_Surround_QVBR-9_1920x1080_5495.mp4" kind="original/media" amg:isPrimary="true" medium="video" duration="1502" framerate="23.97"/>
```

Comment:
can we consider ifPrimary would be assumed to be true if not provided explictly. ??

Reply:
If the media resource count is one then this can be done. If the count is more than 1 can we make sure that only one media resource has isPrimary true or not provided, reset all false?

Reply:
@harshith-d Indeed most of our customers have a single media resource url, so we can add a flag called isSecondary:True as well, where in  case of multiple video its demarcated very clearly.